### PR TITLE
Update UserAuth tests

### PR DESCRIPTION
**Pull request by AndriyShevchuk**
Add tests to check if SDK automatically refreshes expired token

Resolves: OLPEDGE-1233
Signed-off-by: ashevchu <ext-andrii.shevchuk@here.com>

### CODE REVIEW

File: @here/olp-sdk-authentication/test/OAuth.test.ts [hunk 1]
```diff
@@ -496,3 +499,44 @@ describe("auth-request-project-scope", () => {
     });
 
 });
+
+describe("expired token refreshing", async () => {
+    const mockedTokenRequester = async (params: any): Promise<Token> => {
+        const mockedToken: Token = {
+            accessToken: `${params.url}-${params.consumerKey}-${params.secretKey}`,
```
Comment (OleksiiZubko) [inline @here/olp-sdk-authentication/test/OAuth.test.ts]: Please, replace word "secret" to something else (veracode)

Comment (AndriyShevchuk) [reply @here/olp-sdk-authentication/test/OAuth.test.ts]: It is our property from UserAuth.ts
```
.tokenRequester({
                url: this.m_apiUrl + "oauth2/token",
                consumerKey: this.m_credentials.accessKeyId,
                secretKey: this.m_credentials.accessKeyScrt,
                scope: this.m_scope
            })
```
Am I supposed to change it in both places?

Comment (OleksiiZubko) [reply @here/olp-sdk-authentication/test/OAuth.test.ts]: can we just remove "-${params.secretKey}" ?

Comment (AndriyShevchuk) [reply @here/olp-sdk-authentication/test/OAuth.test.ts]: Sure. Done

**Pull request merged**